### PR TITLE
Integrate recalculate filters

### PR DESCRIPTION
**Pull request by moorscode**
Integrated recalculation filters (WordPress SEO 3.1).
Also applied Yoast code styles and added documentation.

Post and Term ACF fields are being added to the content in JavaScript and to the recalculation algorithms.

What does not work yet: wysiwyg ACF fields are not updated in JavaScript yet.
This will be a new issue.

### CODE REVIEW

File: yoast-acf-analysis.php [hunk 1]
```diff
@@ -3,62 +3,219 @@
...
+					$output .= ' ' . $field;
+					break;
+
+				case 'array':
+					if ( isset( $field['sizes']['thumbnail'] ) ) {
+						// Put all images in img tags for scoring.
+						$alt   = ( isset( $field['alt'] ) ) ? $field['alt'] : '';
+						$title = ( isset( $field['title'] ) ) ? $field['title'] : '';
+
+						$output .= ' <img src="' . esc_url( $field['sizes']['thumbnail'] ) . '" alt="' . esc_attr( $alt ) . '" title="' . esc_attr( $title ) . '" />';
```
Comment (jdevalk) [inline yoast-acf-analysis.php]: Why put the `title` here too? Don't see any reason for that.

File: yoast-acf-analysis.php [hunk 2]
```diff
@@ -3,62 +3,219 @@
...
 	}
-	
-	public function require_yoast() { ?>
-		<div class="error"><p>ACF Yoast Analysis requires Yoast SEO 3.0+ to be installed and activated.</p></div><?php
+
+	/**
+	 * Notify that we need WordPress SEO to be installed and active.
+	 */
+	public function wordpress_seo_requirements_not_met() {
+		$message = __( 'ACF Yoast Analysis requires Yoast SEO 3.0+ to be installed and activated.', 'yoast-acf-analysis' );
```
Comment (jdevalk) [inline yoast-acf-analysis.php]: I'd say 3.1+.

File: yoast-acf-analysis.php [hunk 3]
```diff
@@ -3,62 +3,219 @@
...
+			if ( ! is_plugin_active( 'advanced-custom-fields/acf.php' ) && ! is_plugin_active( 'advanced-custom-fields-pro/acf.php' ) ) {
+				add_action( 'admin_notices', array( $this, 'acf_not_active_notification' ) );
 				$deactivate = true;
 			}
-			
-			// Yoast
-			if(!is_plugin_active('wordpress-seo/wp-seo.php') && !is_plugin_active('wordpress-seo-premium/wp-seo-premium.php')) {
-				add_action('admin_notices', array($this, 'require_yoast'));
+
+			// WordPress SEO
```
Comment (jdevalk) [inline yoast-acf-analysis.php]: Yoast SEO for WordPress

File: yoast-acf-analysis.php [hunk 4]
```diff
@@ -3,62 +3,219 @@
...
-	
+
+	/**
+	 * Add notifications to admin if plugins ACF or WordPress SEO are not present.
+	 */
 	public function admin_init() {
-		$this->plugin_data = get_plugin_data(dirname(__FILE__));
-		
+
 		// Require ACF and Yoast
```
Comment (jdevalk) [inline yoast-acf-analysis.php]: Yoast SEO for WordPress

File: yoast-acf-analysis.php [hunk 5]
```diff
@@ -3,62 +3,219 @@
  * Plugin Name: Yoast ACF Analysis
  * Plugin URI: https://forsberg.ax
  * Description: Adds the content of all ACF fields to the Yoast SEO score analysis.
- * Version: 1.0.0
+ * Version: 1.1.0
  * Author: Marcus Forsberg
```
Comment (jdevalk) [inline yoast-acf-analysis.php]: & Team Yoast, leave the link below :)

**Comment by jdevalk** [comment]
You'll want to remove the last line of `readme.txt` too.

**Comment by jcomack** [comment]
CR and Acceptance done :+1:

**Pull request merged**